src/nebula-hooks/use-announce-and-translations.ts

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect } from '@nebula.js/stardust';
+import registerLocale from '../locale/src';
+import { Translator } from '../types';
+
+// eslint-disable-next-line no-shadow
+enum AnnouncerElements {
+  FIRST = 'first-announcer-element',
+  SECOND = 'second-announcer-element',
+}
+
+type AnnounceArgs = {
+  keys: Array<string | Array<string>>;
+  shouldBeAtomic: boolean;
+  politeness: string;
```

Comment:
how about `politeness: 'polite' | 'assertive' | 'off';`? it is more specific.

Reply:
yes, and they should be optional as well

Reply:
RIght